test/e2e/framework/profile_gatherer.go

```diff
@@ -124,67 +153,53 @@ func gatherProfileOfKind(profileBaseName, kind string) error {
 // finish before the parent goroutine itself finishes, we accept a sync.WaitGroup
 // argument in these functions. Typically you would use the following pattern:
 //
-// func TestFooBar() {
+// func TestFoo() {
 //		var wg sync.WaitGroup
 //		wg.Add(3)
-//		go framework.GatherApiserverCPUProfile(&wg, "doing_foo")
-//		go framework.GatherApiserverMemoryProfile(&wg, "doing_foo")
+//		go framework.GatherCPUProfile("kube-apiserver", "before_foo", &wg)
+//		go framework.GatherMemoryProfile("kube-apiserver", "before_foo", &wg)
 //		<<<< some code doing foo >>>>>>
-//		go framework.GatherApiserverCPUProfile(&wg, "doing_bar")
-//		<<<< some code doing bar >>>>>>
+//		go framework.GatherCPUProfile("kube-scheduler", "after_foo", &wg)
 //		wg.Wait()
 // }
 //
 // If you do not wish to exercise the waiting logic, pass a nil value for the
 // waitgroup argument instead. However, then you would be responsible for ensuring
-// that the function finishes.
+// that the function finishes. There's also a polling-based gatherer utility for
+// CPU profiles available below.
 
-func GatherApiserverCPUProfile(wg *sync.WaitGroup, profileBaseName string) {
-	GatherApiserverCPUProfileForNSeconds(wg, profileBaseName, DefaultCPUProfileSeconds)
+func GatherCPUProfile(componentName string, profileBaseName string, wg *sync.WaitGroup) {
+	GatherCPUProfileForNSeconds(componentName, profileBaseName, DefaultCPUProfileSeconds, wg)
 }
 
-func GatherApiserverCPUProfileForNSeconds(wg *sync.WaitGroup, profileBaseName string, n int) {
+func GatherCPUProfileForNSeconds(componentName string, profileBaseName string, N int, wg *sync.WaitGroup) {
```

Comment:
s/ForNSeconds/ForSeconds/
s/N/seconds/

?

Reply:
BTW - this function doesn't seem to be called from anywhere externally - so let's make that private one

Reply:
> s/ForNSeconds/ForSeconds/ s/N/seconds/

Done

> BTW - this function doesn't seem to be called from anywhere externally - so let's make that private one

I'd prefer keeping it public - to not force users of this utility (which although is just us currently) to only be able to use polling-based gatherer. Wdyt?